frontend/online-shop/src/styles/ProductDetailViewStyle.tsx

```diff
@@ -0,0 +1,45 @@
+import { makeStyles } from "@material-ui/core";
+
+export const useProductDetailViewStyles = makeStyles({
+  everything: {
+    textAlign: "center",
+    marginTop: "5%",
+    display: "block",
+  },
+  title: {
+    fontSize: "20px",
+    marginTop: "1%",
+    marginBottom: "0.5%",
+    display: "block !important",
+  },
+  image_product: {
```

Comment:
Use either cammel case `imageProduct` or underscore case `image_product` but keep it uniform